setup.py

```diff
@@ -29,7 +29,7 @@
 INSTALL_REQUIRE = [
     "torch>=1.11",
     "optimum~=1.25",
-    "transformers>=4.36,<4.52",
+    "transformers>=4.36,<4.52.3",
```

Comment:
I think we should include patch release by default so would replace with

```suggestion
    "transformers>=4.36,<4.53",
```